fs/grub/shared.h

```diff
@@ -371,6 +371,9 @@ extern char *grub_scratch_mem;
 #define strcpy grub_strcpy
 #endif /* WITHOUT_LIBC_STUBS */
 
+#define SECTOR_SIZE 512
+#define MIN(x,y) ((x)<(y)?(x):(y))
```

Comment:
nit: If this exists elsewhere, use it. Otherwise I'd suggest to move this into the file where it's used, or, to remove the macro altogether, as our use case can be dumped down into a single if statement along the lines of:

```
int bytes_available = SECTOR_SIZE - byte_offset;
int bytes_to_copy = byte_len;
if (bytes_to_copy > bytes_available) {
    bytes_to_copy = bytes_available;
}
```

(which is hopefully also self-documenting)

Reply:
Moved macro to the source file.